collectors/gsuite/gsuite_collector.js

```diff
@@ -0,0 +1,117 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ *
+ * Auth0 collector class.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const moment = require('moment');
+const request = require('request');
+const parse = require('@alertlogic/al-collector-js').Parse;
+const { google } = require('googleapis');
+const { auth } = require('google-auth-library');
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+
+const typeIdPaths = [
+    { path: ['kind'] }
+];
+
+const tsPaths = [
+    { path: ['id','time'] }
+];
+
+function  listLoginEvents(auth, params, callback) {
+    const service = google.admin({version: 'reports_v1', auth});
+    params['userKey'] = 'all';
+    params['applicationName'] = 'login';
+    service.activities.list(params, (err, res) => {
```

Comment:
In this current state, this call will list all log-in events from the default start time as it's an optional parameter, I believe it will be set to 180 days prior to the API call. The timestamp information from the collection state should be used here to prevent re-collection and limit the returned logs to a reasonable timeframe

Reply:
Ah, I see it's on line 62, my bad this time.

Reply:
@JDragovichAlertLogic How do you envision pagination to be handled? Do you want to loop the API call till data expiry?

Reply:
You could loop over the api calls until there is no longer a `nextPageToken`. That can be done in a single lambda invocation.